src/styles/blocks/header.scss

```diff
@@ -0,0 +1,91 @@
+.header {
+  @include page-grid;
+
+  box-sizing: border-box;
+  overflow: hidden;
+  height: 100vh;
```

Comment:
This height is doing the size of header on mobile be higher than expected. Try to adjust the height of header or the padding of the features section.

![image](https://github.com/mate-academy/Kickstarter/assets/37789845/44112c00-86eb-4917-8dec-7feaa949e9d2)